src/tests/mocks/service_transactions_statistics_mock.js

```diff
@@ -0,0 +1,617 @@
+const nock = require('nock')
```

Comment:
This file breaks lint/ CI on quite a few rules (mostly `"` -> `'`). We don't necessarily need to lint test files but as this is currently being picked up it will need to be addressed. Changing double quotes to single should do it.